packages/calcite-components/src/components/alert/alert.tsx

```diff
@@ -201,12 +208,6 @@ export class Alert implements OpenCloseComponent, LoadableComponent, T9nComponen
   }
 
   render(): VNode {
-    numberStringFormatter.numberFormatOptions = {
```

Comment:
This looks good but there are other components doing this same thing. Should we update them as well?

Reply:
Yeah I was thinking the same thing. I'll start doing that after I finish some of my other issues for this milestone.

Reply:
FWIW this is the only one causing an actual bug that I'm aware of because of the delay. See the repro sample:
https://codepen.io/benesri/pen/YzdodgJ